Header header

VehicleState truck_head
VehicleState[] trailers
